lib/lte_link_control/lte_lc.c

```diff
@@ -184,19 +184,10 @@ static const char *const system_mode_params[] = {
 	const char *const lwm2m_ant_cfg[] = {
 			"AT%XMAGPIO=1,1,1,7,1,746,803,2,698,748,"
 			"2,1710,2200,3,824,894,4,880,960,5,791,849,"
-			"7,1565,1586",
-			"AT%XCOEX0=1,1,1565,1586"};
-#elif defined(CONFIG_BOARD_NRF9160DK_NRF9160NS)
-	const char *const lwm2m_ant_cfg[] = {
```

Comment:
Since this is removed, the outdated line:
https://github.com/nrfconnect/sdk-nrf/blob/eb068b70e303f8c5f404833f160641575743c763/lib/lte_link_control/lte_lc.c#L553
should be also removed, to not generate confusion.

Reply:
Done. Also did some other changes to improve readability.
The MAGPIO command will now be sent for Thingy:91 if `CONFIG_BSD_LIBRARY_SYS_INIT=n`, and not only if LwM2M carrier library is enabled. This catches more cases where this command is required in the link controller.